tensorflow/python/ops/math_ops.py

```diff
@@ -100,9 +100,113 @@
 from tensorflow.python.util.tf_export import tf_export
 
 # Aliases for some automatically-generated names.
-linspace = gen_math_ops.lin_space
```

Comment:
> Redirects to this url : https://kite.com/python/docs/keras.backend.control_flow_ops.array_ops.gen_math_ops

> I noticed there’s a persistent [F] symbol at the end of every line

>I think we should find a way to figure this out : [\:/ =/= /:\]

>I suggest we could try to merge that into a container? Something like this maybe : \<[/:\ =♡︎= \:/]>/

>Again, it’s only a suggestion

Reply:
I have no idea what you mean. Could you elaborate?

Reply:
That does not look like a valid review

Reply:
Doesn't look

> I have no idea what you mean. Could you elaborate?

I think